src/index.js

```diff
@@ -0,0 +1,197 @@
+"use strict";
+
+
+// let map;
+// let service;
+// function callback(place, status) {
+//     if (status == google.maps.places.PlacesServiceStatus.OK) {
+//         createMarker(place);
+//     }
+// }
+// let place= //input city
+// let config = {
+//     method: 'get',
+//     url: 'https://maps.googleapis.com/maps/api/place/textsearch/json?query=place&key=YOUR_API_KEY'
+
+// };
+
+// axios(config)
+
+// .then((response)=>{
+//     const place = response.data[results][0];
+//     request={
+//     placeId: place.place_id
+//     }
+//     service = new google.maps.places.PlacesService(map);
+//     service.getDetails(request, callback);
+// });
+// .then((response)=>{
+//     PlacePhoto=response.photos[0];
+//     URL=PlacePhoto.getUrl();
+//     parameters={
+//     maxwidth=400,
+//     key=YOUR_API_KEY
+//     }
+// return axios.get(URL, parameters);
+// });
+const state = {
+    degree:60,
+    isF: true,
+    cityName: 'Poppy City, USA',
+    
+};
+//let theText = myTextInput.value;
+const showCityName=()=>{
+    const cityContainer = document.getElementById("cityDisplay");
+    const city= document.getElementById('name')
+    const cityName= city.value;
+    cityContainer.textContent = `Showing weather for: ${cityName}`;
+}   
+const changeToC=()=>{
+    const degreeCountContainer = document.getElementById("degree");
+    if (state.isF){
+        state.isF=false;
+        degreeCountContainer.textContent = Math.floor((state.degree -32)/1.8);
+    }
+
+}
+const changeToF=()=>{
+    const degreeCountContainer = document.getElementById("degree");
+    if (! state.isF){
+        state.isF=true;
+    }
+    degreeCountContainer.textContent=state.degree;
+}
+const changeTemp=(e)=> { //refactor 2 functions together to change temp
+    const degreeCountContainer = document.getElementById("degree");
+    e.target.id ==='plus'? state.degree++: state.degree--;
+    degreeCountContainer.textContent=state.degree;
```

Comment:
Always good practice to add spaces around operators. 

```suggestion
    e.target.id === 'plus'? state.degree++ :  state.degree--;
    degreeCountContainer.textContent = state.degree;
```